src/java.base/share/classes/java/util/concurrent/ConcurrentSkipListSet.java

```diff
@@ -189,14 +189,9 @@ public ConcurrentSkipListSet<E> clone() {
      * contains more than {@code Integer.MAX_VALUE} elements, it
      * returns {@code Integer.MAX_VALUE}.
      *
-     * <p>Beware that, unlike in most collections, this method is
-     * <em>NOT</em> a constant-time operation. Because of the
-     * asynchronous nature of these sets, determining the current
-     * number of elements requires traversing them all to count them.
-     * Additionally, it is possible for the size to change during
-     * execution of this method, in which case the returned result
-     * will be inaccurate. Thus, this method is typically not very
-     * useful in concurrent applications.
+     * <p>It is possible for the size to change during execution of this method,
```

Comment:
@jaikiran Realized the paragraph fell away so I re-added it. Reapproval needed?

Reply:
Hello Viktor, the new review process in the JDK project marks the previous review as stale if a change is done after the approval. Re-review is mandated in such cases. I've now approved the latest change which looks good to me.

Reply:
Thanks @jaikiran :+1: